### ISSUE TITLE
utils.py [output_mode = regression]

### ISSUE THREAD

**Issue by vr25** [ThilinaRajapakse/pytorch-transformers-classification#34]
Hi,

I am trying to predict a score (float) using regression and I was wondering what changes need to be made in lines 254 and 258 in [utils.py](https://github.com/ThilinaRajapakse/pytorch-transformers-classification/blob/master/utils.py)

Thanks!

**Comment by vr25** [comment]
Also, since it uses BinaryClassificationProcessor, it creates the logits tensor with size of [10,2] (say, my input file has 10 texts). After view(-1), logits becomes size=20 but the labels are still size=10. Hence, MSELoss() returns the following size mismatch error:

![image](https://user-images.githubusercontent.com/22553367/69378550-9431ef80-0c7c-11ea-8098-8e237b1204ae.png)

Thanks!

**Comment by ThilinaRajapakse** [comment]
Set this in the args dict. 

`'output_mode': 'regression'`

**Comment by vr25** [comment]
Alright, sure, thanks, I'll do that and see if it works.

Also, I was wondering if you could take a look at [this](https://github.com/huggingface/transformers/issues/1907) [I am trying to do domain fine-tuning]

**Comment by vr25** [comment]
On setting:
'output_mode': 'regression'
it still gives the following error:

![image](https://user-images.githubusercontent.com/22553367/69432206-3bf3ff80-0d07-11ea-91c5-105a02433dd1.png)
